### PR TITLE
Adding Richter Slides

### PR DESCRIPTION
**Pull request by ZoeRichter**
Adding in:

My two slides
1 image, titled mmr-chalk-river
my acknowledgements
my citation in the bibliography

### CODE REVIEW

File: 2019-09-05-group.bib [hunk 1]
```diff
@@ -34,4 +34,13 @@ @techreport{eldred_dakota_2010
 	year = {2010},
 	doi = {10.2172/991842},
 	file = {Eldred et al. - 2010 - DAKOTA  a multilevel parallel object-oriented fra.pdf:/Users/gwenchee/Zotero/storage/Q5XJE57U/Eldred et al. - 2010 - DAKOTA  a multilevel parallel object-oriented fra.pdf:application/pdf}
-}
```
Comment (samgdotson) [inline 2019-09-05-group.bib:37]: What is this? Make sure you didn't accidentally delete something important.

File: richter.tex [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  \column[t]{5cm}
+  Microreactors can fulfill energy needs in special use cases, or in
+  areas with a greater population.
+  \column[t]{5cm}
+    \begin{figure}[htbp!]
+    \begin{center}
+    \includegraphics[height=4cm]{./images/mmr-chalk-river}
+    \end{center}
+    \caption{A Diagram of the proposed MMR for the Chalk River project. \cite{mmr-chalk-river}.}
+    \label{fig:kittenfigure}
```
Comment (samgdotson) [inline richter.tex]: Is this label accurate?
maybe `fig:mmr`

**Review by samgdotson — CHANGES_REQUESTED** [review]

File: 2019-09-05-group.bib (hunk 1 above)
Comment (ZoeRichter) [reply 2019-09-05-group.bib:37]: That is the end bracket that holds the entire citation.  If you look, they are all formatted to have that bracket in its own line.

File: richter.tex (hunk 2 above)
Comment (ZoeRichter) [reply richter.tex]: oops!  Got it, thanks!

File: 2019-09-05-group.bib (hunk 1 above)
Comment (ZoeRichter) [reply 2019-09-05-group.bib:37]: Sorry - I should clarify that I checked in the file, and that bracket is still there.  I think it shows thi deletion, and then, immediately below it, it shows it as an addition.

**Pull request merged**